rpmlint/checks/BashismsCheck.py

```diff
@@ -0,0 +1,31 @@
+import stat
+import subprocess
+
+from rpmlint.checks.AbstractCheck import AbstractFilesCheck
+
+
+class BashismsCheck(AbstractFilesCheck):
+    def __init__(self, config, output):
+        super().__init__(config, output, r'.*')
+
+    def check_file(self, pkg, filename):
+        root = pkg.dirName()
+        pkgfile = pkg.files()[filename]
+        fullpath = root + filename
+
+        if not (stat.S_ISREG(pkgfile.mode) and
+                pkgfile.magic.startswith('POSIX shell script')):
```

Comment:
Isn't 'POSIX shell script' everything that ends with .sh ?
We have to really implement queries for the pkg.py something like pkg.configs, pkg.headers, pkg.binaries,...

Reply:
> Isn't 'POSIX shell script' everything that ends with .sh ?

Nope:
```
$ file /usr/bin/* | grep 'POSIX shell script' | grep -v '\.sh' | wc -l
500
```

Reply:
Interesting, i am not disputing that but I created sample .sh file where I set shebang to point to various areas:
 * bash ` Bourne-Again shell script, ASCII text executable`
 * env sh `a /usr/bin/env sh script, ASCII text executable`
 * /bin/sh `POSIX shell script, ASCII text executable`
 * /usr/bin/sh `a /usr/bin/sh script, ASCII text executable`

So probably it is correct right?

Reply:
> So probably it is correct right?

I guess so.